docs/gathered-data.md

```diff
@@ -979,6 +964,23 @@ gathers the alerts that have been silenced.
   - 4.10+
 
 
+## StorageCluster
+
+collects `storageclusters.ocs.openshift.io` resources
+from Openshift Data Foundation Stack.
+
+API Reference:
+
+	https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/storagecluster_types.go
+
+* Location in archive:
+	 - 4.11.z : config/storage/<namespace>/<name>.json
+	 - 4.12+  : config/storage/<namespace>/storageclusters/<name>.json
+* Id in config: clusterconfig/openshift_storage
```

Comment:
Also here, how it is going to look when whe change the config key, 'cause we change the Gather name?

Reply:
changing config key is probably OK in my opinion.

Reply:
I'd do the same as for the file path.